code/game/machinery/atmoalter/meter.dm

```diff
@@ -7,7 +7,7 @@
 	anchored = 1.0
 	power_channel = ENVIRON
 	var/frequency = 0
-	var/id
+	id

```

Comment:
:thinking:

Reply:
same as `id = null`

Reply:
Yeah, but just remove it if it's unmodified from the parent.